logicrunner/builtin/contract/migrationdaemon/migrationdaemon.go

```diff
@@ -17,17 +17,54 @@
 package migrationdaemon
 
 import (
+	"fmt"
+	"math/big"
+
 	"github.com/insolar/insolar/insolar"
 	"github.com/insolar/insolar/logicrunner/builtin/foundation"
+	"github.com/insolar/insolar/logicrunner/builtin/proxy/deposit"
+	"github.com/insolar/insolar/logicrunner/builtin/proxy/member"
+	"github.com/insolar/insolar/logicrunner/builtin/proxy/migrationadmin"
+	"github.com/insolar/insolar/logicrunner/builtin/proxy/wallet"
 )
 
+const CONVERSION = "10"
+
 // MigrationDaemonContract make migration procedure.
 type MigrationDaemon struct {
 	foundation.BaseContract
 	IsActive              bool
 	MigrationDaemonMember insolar.Reference
 }
 
+// Migration methods.
+type DepositMigrationResult struct {
+	Reference string `json:"memberReference"`
+}
+
+//Call internal function migration admin from api.
```

Comment:
Codestyle: comment on public var should started with name like:  
// DepositMigrationCall calls internal function migration admin from api.

Reply:
Fix